Library/Homebrew/cask/audit.rb

```diff
@@ -270,6 +273,83 @@ def bad_osdn_url?
       bad_url_format?(/osd/, [%r{\Ahttps?://([^/]+.)?dl\.osdn\.jp/}])
     end
 
+    def extract_url(url)
+      url.to_s.gsub(%r{.*"([a-z0-9]+://[^"]+)".*}m, '\1')
+    end
+
+    def remove_non_ascii(string)
```

Comment:
Why is this necessary?

Reply:
I just follow the logic from `style`. I remove it.